kani-compiler/src/codegen_cprover_gotoc/codegen/statement.rs

```diff
@@ -628,6 +628,23 @@ impl<'tcx> GotocCtx<'tcx> {
                         .assign(self.codegen_rvalue(r), Location::none())
                 }
             }
+            StatementKind::Deinit(place) => {
+                // From rustc doc: "This writes `uninit` bytes to the entire place."
+                // Thus, we assign nondet() value to the entire place.
+                let dst_mir_ty = self.place_ty(place);
+                let dst_type = self.codegen_ty(dst_mir_ty);
+                let layout = self.layout_of(dst_mir_ty);
+                if layout.is_zst() || self.ignore_var_ty(dst_mir_ty) {
+                    // We ignore assignment for all zero size types
+                    // Ignore generators too for now:
+                    // https://github.com/model-checking/kani/issues/416
+                    Stmt::skip(Location::none())
```

Comment:
No location available?

Reply:
I can probably fix that. Let me see. I just took this code from the way we handle assignment.